src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/OffersController.java

```diff
@@ -109,9 +109,8 @@ private void assertActionIsPermittedFor(Claim claim, MadeBy party, String author
         if (party.equals(MadeBy.CLAIMANT)) {
             authorisationService.assertIsSubmitterOnClaim(claim, userDetails.getId());
         }
-        if (party.equals(MadeBy.DEFENDANT)) {
-            authorisationService.assertIsDefendantOnClaim(claim, userDetails.getId());
-        }
+        // We have no way to check if a user is a defendant on the claim other than CCD allowing the save
```

Comment:
In this case we should just have a single `isPartyOnClaim` or `hasAccessToClaim` method in the auth service. Please update this.

Reply:
IMO we can't do any of these checks and we should just delete this service, its up to CCD to do authorisation now. but I think this can be deleted later

Reply:
Until we remove the service and let CCD do authorisation, can you please rename the method so that it's not related to claimant only?

Reply:
removed method as I can't see a way where it still makes sense